### DISCUSSION TITLE
[Question] Can I customize the default page.pubsub implementation?

### DISCUSSION THREAD

**Discussion by luozhouyang** [flet-dev/flet#5732] [Q&A]
According to the official documents, I am not sure how flet implement pubsub in detail. Does flet support customize pubsub implementations? To be more specific, I want to dpeloy flet apps in a k8s cluster with more than three pods, how does flet default pubsub work with this distributed scene?

**Comment by FeodorFitsner** [comment]
I assume you need to replace `PubSubHub` (`packages/flet/src/flet/pubsub/pubsub_hub.py`) with your own implementation.

For web it's currently being instantiated in here: https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-web/src/flet_web/fastapi/flet_app_manager.py#L37-L47

I'd think to give someone an ability to plug into creation of PubSubHub we need a factory method in `FletAppManager`. Does it make sense?
